coalib/bears/requirements/PackageRequirement.py

```diff
@@ -119,3 +120,19 @@ def multiple(cls, *args):
                 else:
                     raise TypeError('Too many elements provided.')
             return set(reqs)
+
+    @classmethod
+    def upload(self):
+        """
+        An upload method which uploads a conda package containing the command
+        to install the corresponding package.
+        """
+        name = self.package
+        if not os.path.exists(name):
+            os.mkdir(name)
+        file = open(name + '/' + name + '.py', 'a')
+        file.write(self.manager + ' ' + self.package)
```

Comment:
Also I think the current file I/O approach can be made better. This is more pythonic and you won't have to close the file descriptor explicitly :
```python
with open(os.path.join('name','name.py'), 'a') as package_file:
    package_file.write("{manager} {package}".format(manager=self.manager, 
                                                    package=self.package))
```